docs/easy_anticheat_configuration.md

```diff
@@ -17,4 +17,4 @@ The EOS plugin provides a method of automating EAC installation as part of the b
     1) If you want to customize the config for EAC integrity tool, you can copy the default config from `Packages/Epic Online Services Plugin for Unity/Editor/Standalone/anticheat_integritytool.cfg` to your desired location, customize it, and set `Path to EAC Integrity Tool Config` to the new file.
```

Comment:
On line 11: should it say "With the plugin installed"?

Reply:
I changed that, but it's phrased that way in configuring_the_eos_plugin.md as well